src/main/java/com/example/authservice/messaging/MessagePublisher.java

```diff
@@ -0,0 +1,26 @@
+package com.example.authservice.messaging;
+
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Date;
+import java.util.UUID;
+
+@RestController
```

Comment:
This should be a regular class without `RestController` and we should also remove the` @RequestBody ` annotation on line 18.

According to Baeldung:
> Spring 4.0 introduced the @RestController annotation in order to simplify the creation of RESTful web services.